NuGet.config

```diff
@@ -2,19 +2,36 @@
 <configuration>
   <packageSources>
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-aspnetcore-tooling -->
+    <add key="darc-int-dotnet-aspnetcore-tooling-7f783df" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-aspnetcore-tooling-7f783df7/nuget/v3/index.json" />
+    <!--  End: Package sources from dotnet-aspnetcore-tooling -->
+    <add key="darc-int-dotnet-extensions-4210e7d" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-extensions-4210e7d1/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-efcore-d078590" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-efcore-d078590d/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-core-setup-0267ad0" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-0267ad09/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-corefx-50a99d5" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-50a99d5d/nuget/v3/index.json" />
```

Comment:
I thought we didn't want these sources?

Reply:
Oh I see, these are then disabled

Reply:
The new thing is that they all stay in nuget.config, but automation disables them in public builds. Also means that nuget.config will continue to grow in size since old feeds don't get removed